### PR TITLE
Created Cards component

### PR DESCRIPTION
**Pull request by anushasunil**
- Card with badges
- Card with dismiss
- Card with text overlay
- Text only card
- Horizontal cards
- Card with shadow

Screenshot-20220207023827

### CODE REVIEW

File: Stylesheets/Components/card.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
+img {
```
Comment (dev-enforced) [inline Stylesheets/Components/card.css]: You can add a class to the image as card-img . No need of selecting the element related to it.So you can take care of it appropriately.

File: Stylesheets/Components/card.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    gap: 10px;
+    padding: 8px;
+}
+
+.strike-off {
+    color: var(--color-error-dark);
+    font-size: 1.4rem;
+}
+
+.ratings img {
```
Comment (dev-enforced) [inline Stylesheets/Components/card.css]: Here also don't select the img element you can pass a class to that image and refer to it as .ratings .card-img

File: Stylesheets/Components/card.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    gap: 1rem;
+}
+
+.horizontal-card .content {
+    flex-grow: 1;
+}
+
+
+.with-shadow {
+    box-shadow:0 0 20px -1px rgba(128, 128, 128, 0.39);
```
Comment (dev-enforced) [inline Stylesheets/Components/card.css]: You can use a variable in the utility class and pass this value there and use it as a variable.
Like --card-box-shadow:0 0 20px -1px rgba(128, 128, 128, 0.39); and then use it here in this class
.with-shadow{
box-shadow:var(--card-box-shadow);
}

**Review by dev-enforced — APPROVED** [review]
The code and output looks really awesome to me!! Good to merge Anusha✔
Suggested some minor changes which you can see and decide accordingly.

File: Stylesheets/Components/card.css (hunk 1 above)
Comment (anushasunil) [reply Stylesheets/Components/card.css]: Oh understood!

File: Stylesheets/Components/card.css (hunk 2 above)
Comment (anushasunil) [reply Stylesheets/Components/card.css]: Yes, I have updated the code. Please check it once again. Thanks!

File: Stylesheets/Components/card.css (hunk 3 above)
Comment (anushasunil) [reply Stylesheets/Components/card.css]: Thankyou for pointing this out, I forgot to declare it as a variable

**Review by simrk — COMMENTED** [review]
Lgtm.

**Pull request merged**